### PR TITLE
Prefix username with project

### PR DESCRIPTION
**Pull request by smlx**
Generate database usernames which are prefixed with project names. This will make it easier to attribute database usage statistics to projects.

### CODE REVIEW

**Comment by smlx** [comment]
This PR builds on top of #2, so the other PR should be reviewed/merged first.

**Review by thom8 — APPROVED** [review]
Nice work, LGTM!

**Review by shreddedbacon — APPROVED** [review]

File: tasks/main.yml [hunk 1]
```diff
@@ -22,14 +23,15 @@
   block:
     - set_fact:
         create_db_name: "{{ namespace | truncate(50, true, '') }}_{{ lookup('password', '/tmp/_name length=5 chars=ascii_letters') }}"
-        create_db_user: "{{ lookup('password', '/tmp/_user length=16 chars=ascii_letters') }}"
+        create_db_user: "{{ namespace | truncate(50, true, '') }}_{{ lookup('password', '/tmp/_user length=16 chars=ascii_letters') }}"
```
Comment (Schnitzel) [inline tasks/main.yml]: this is gonna fail very very very hard: https://dba.stackexchange.com/questions/147857/username-length-restriction-in-different-mysql-environments 
to be safe we should have mysql usernames max 16 chars (or maybe 32 characters, if we have mysql version `5.7.8` everywhere)

**Review by Schnitzel — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (smlx) [reply tasks/main.yml]: ah thanks @Schnitzel. I only checked (and tested against) [mariadb](https://mariadb.com/kb/en/library/identifier-names/#maximum-length), which has a limit of 80.

Comment (smlx) [reply tasks/main.yml]: RDS still support mysql 5.5(!), so a limit of 16 seems prudent.

**Comment by smlx** [comment]
This now builds on #5, which should be merged first.

**Review by thom8 — APPROVED** [review]
LGTM

**Pull request merged**